toss/resources/default_cfg.toml

```diff
@@ -21,8 +21,8 @@ activate_event = true                   # Event configuration (False = no events
 number_of_maneuvers = 0                 # Maximum number of possible manuevers during mission.
 
 [optimization]          
-population_size = 7                     # Number of chromosomes per island for the pygmo-based optimization.
-number_of_generations = 1               # Number of generations for evolutionary optimization process.
+population_size = 20                     # Number of chromosomes per island for the pygmo-based optimization.
```

Comment:
if we use BFE now I would suggest increasing the popsize to some multiple of number of threads (40) you want to use. maybe 80?